python/tk_maya_playblast/playblast.py

```diff
@@ -86,12 +90,16 @@ def __createPlayblast(self, **overridePlayblastParams):
                     resultPlayblastPath = pm.playblast( **playblastParams )
                     playblastSuccessful = True
                 except RuntimeError, e:
-                    result = QtGui.QMessageBox.critical(None, u"Playblast Error",
-                                                        "%s\n\n... or just close your QuickTime player, and Retry." % unicode(e),
-                                                        QtGui.QMessageBox.Retry | QtGui.QMessageBox.Abort)
-                    if result==QtGui.QMessageBox.Abort:
-                        self._app.log_error("Playblast aborted")
-                        return
+                    if not os.path.exists(localPlayblastPath):
+                        result = QtGui.QMessageBox.critical(None, u"Playblast Error",
+                                                            "%s" % unicode(e),
+                                                            QtGui.QMessageBox.Retry | QtGui.QMessageBox.Abort)
+                        if result==QtGui.QMessageBox.Abort:
+                            self._app.log_error("Playblast aborted")
+                            return
+                    else:
+                        viewer_issues = True
+                        playblastSuccessful = True
```

Comment:
Switch around the `if...else`, also tidy up args and typical coding conventions since the lines will be under your name

```suggestion
                    if os.path.exists(localPlayblastPath):
                        viewer_issues = True
                        playblastSuccessful = True
                    else:
                        result = QtGui.QMessageBox.critical(None, u"Playblast Error",
                                                            unicode(e),
                                                            QtGui.QMessageBox.Retry | QtGui.QMessageBox.Abort)
                        if result == QtGui.QMessageBox.Abort:
                            self._app.logger.exception("Playblast aborted")
                            return
```